install/installer/dcs_installer

```diff
@@ -136,7 +136,8 @@ echo "***INFO: untarring build file $DCS_BUILD_FILE"
 tar -xzf $DCS_BUILD_FILE
 
 # get the dcs install directory name which is imbedded in the tar file
-DCS_DIR=$(tar -tf $DCS_BUILD_FILE | grep LICENSE | sed -e "s@\(^.*\)/LICENSE.txt@\1@")
+DCS_DIR=$(tar -tf $DCD_BUILD_FILE | grep --max-count=1 bin | sed -e "s@\(^.*\)/bin/@\1@")
```

Comment:
Yes, it is. Thanks Dave!

On Tue, Nov 24, 2015 at 2:37 PM, DaveBirdsall <notifications@github.com>
wrote:

> In install/installer/dcs_installer
> <https://github.com/apache/incubator-trafodion/pull/187#discussion_r45805519>
> :
>
> > @@ -136,7 +136,8 @@ echo "***INFO: untarring build file $DCS_BUILD_FILE"
> >  tar -xzf $DCS_BUILD_FILE
> >
> >  # get the dcs install directory name which is imbedded in the tar file
> > -DCS_DIR=$(tar -tf $DCS_BUILD_FILE | grep LICENSE | sed -e "s@\(^.*\)/LICENSE.txt@\1@")
> > +DCS_DIR=$(tar -tf $DCD_BUILD_FILE | grep --max-count=1 bin | sed -e "s@\(^.*\)/bin/@\1@")
>
> Is there a typo here? "DCD..." as opposed to "DCS..."
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-trafodion/pull/187/files#r45805519>.
>



-- 
Thanks,

Amanda Moran